lib/constants.go

```diff
@@ -699,10 +699,25 @@ type DeSoParams struct {
 	// before they are jailed.
 	DefaultJailInactiveValidatorGracePeriodEpochs uint64
 
+	// DefaultFeeBucketRateMultiplierBasisPoints is the rate of growth of the fee bucket ranges. The multiplier is given
+	// as basis points. For example a value of 1000 means that the fee bucket ranges will grow by 10% each time.
+	DefaultFeeBucketRateMultiplierBasisPoints uint64
+
+	// DefaultFailingTransactionBMFRateBasisPoints is the default rate of the failing transaction fee, in basis points,
+	// that is used to in BMF calculations. E.g. a value of 2500 means that 25% of the failing transaction's fee is used
+	// in BMF calculations.
+	DefaultFailingTransactionBMFRateBasisPoints uint64
+
 	ForkHeights ForkHeights
 
 	EncoderMigrationHeights     *EncoderMigrationHeights
 	EncoderMigrationHeightsList []*MigrationHeight
+
+	// The maximum aggregate number of bytes of transactions included in the PoS mempool.
+	MaxMempoolPosSizeBytes uint64
+
+	// MempoolBackupTimeMilliseconds is the frequency with which pos mempool persists transactions to storage.
+	MempoolBackupTimeMilliseconds uint64
```

Comment:
This should definitely be a flag not a ParamUpdater param.

Reply:
It should still be in DeSo params though, no? Just not in GlobalParams.